atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/CqlCluster.java

```diff
@@ -57,22 +73,34 @@ public void close() throws IOException {
     }
 
     public Map<InetSocketAddress, RangeSet<LightweightOppToken>> getTokenRanges(String tableName) {
-        try (CqlSession session = new CqlSession(cluster.connect())) {
+        try (CqlSession session = createSessionWithRetry()) {
             return new TokenRangeFetcher(session, namespace, cassandraServersConfig).getTokenRange(tableName);
```

Comment:
should be fine to have one session per call as these calls are serial

Reply:
ack, won't do that for now as combining the calls will be a slightly involved refactor.